src/hotspot/share/gc/parallel/psParallelCompact.cpp

```diff
@@ -126,73 +126,82 @@ ParallelCompactData::RegionData::dc_claimed = 0x8U << dc_shift;
 const ParallelCompactData::RegionData::region_sz_t
 ParallelCompactData::RegionData::dc_completed = 0xcU << dc_shift;
 
+bool ParallelCompactData::RegionData::is_clear() {
```

Comment:
Do we call this anywhere outside #ifdef ASSERT?

Reply:
No, it's only for assert. The name seems useful for non-verification, so I left it there.